developing/vcs/master-to-main.md

```diff
@@ -0,0 +1,89 @@
+# [Source Control](./README.md) / Master to Main branch rename
+
+These instructions will help moving repos from using the default branch name `master` and modify it to be
+`main`.
+
+## Instructions to change default branch name
+
+To update the branch name use the npm tool [lgarron/main-branch](https://github.com/lgarron/main-branch):
+
+```sh
+npm install -g main-branch
+main-branch trussworks/master-to-main-test replace master main
+```
+
+Follow up with local modifications:
+
+```sh
+cd git-repo-name
+git pull origin main
+git checkout main
+git branch --delete master
+git branch --set-upstream-to=origin/main main
+git symbolic-ref HEAD refs/heads/main
+git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main
+git fetch --all --prune
+```
+
+Test that the `master` branch has been removed by running this command and getting an error:
```

Comment:
Given that this is a test, what should the output be that I'm expecting.

Reply:
`getting an error` is the test that validates it.

Reply:
I think I'd want to know what error I'm expecting or how I know things have gone off the rails :P

Reply:
It's just a shell error. I guess I could write `running this command and getting a shell error` if that makes it more clear.